File: app/src/main/java/com/slidely/photoframe/features/preferences/PreferencesActivity.java
package com.slidely.photoframe.features.preferences;

import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.slidely.photoframe.R;
import com.slidely.photoframe.features.Utils;

public class PreferencesActivity extends AppCompatActivity {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_preferences);
        getSupportFragmentManager()
                .beginTransaction()
                .add(R.id.content, new PreferencesFragment())
                .commitAllowingStateLoss();
        Utils.hideBars(this);
    }
}

File: app/src/main/java/com/k4dima/photoframe/features/preferences/PreferencesActivity.kt
package com.k4dima.photoframe.features.preferences

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.transaction
import com.k4dima.photoframe.R

class PreferencesActivity : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_preferences)
        supportFragmentManager.transaction {
            add(R.id.content, PreferencesFragment())
        }
    }
}